install/fedora-installation-guide.md

```diff
@@ -2,60 +2,31 @@
 
 > **Notes:**
 >
-> - Kata Containers packages are available for Fedora\* versions **26** and **27** (currently `x86_64` only).
+> - Kata Containers packages are available for [Fedora\*](https://fedoraproject.org)
+>   versions **26** and **27** (currently `x86_64` only).
 >
 > - If you are installing on a system that already has Clear Containers or `runv` installed,
 >   first read [the upgrading document](../Upgrading.md).
 >
-> - This install guide is specific to integration with Docker.  If you want to use Kata
->   Containers through Kubernetes, see the "Developer Guide" [here](https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#if-you-want-to-run-kata-containers-with-kubernetes).
+> - If you do not wish to copy or type all these instructions by hand, you can use either use the
+>   [`kata-manager`](https://github.com/kata-containers/tests/blob/master/cmd/kata-manager/kata-manager.sh)
+>   script to install the packaged system including your chosen container manager. Alternatively, you can generate
+>   a runnable shell script from individual documents using the
+>   [`kata-doc-to-script`](https://github.com/kata-containers/tests/blob/master/.ci/kata-doc-to-script.sh) script.
 
-This step is only required in case Docker is not installed on the system.
-1. Install the latest version of Docker with the following commands:
+1. Install the Kata Containers components with the following commands:
 
-```bash
-$ sudo dnf -y install dnf-plugins-core
-$ sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
-$ sudo dnf makecache
-$ sudo dnf -y install docker-ce
-```
+   > **Note:** This installation channel is not secure since the repository currently
+   > redirects download URLs to `http`.
 
-For more information on installing Docker please refer to the
-[Docker Guide](https://docs.docker.com/engine/installation/linux/fedora)
+   ```bash
+   $ source /etc/os-release
+   $ sudo dnf -y install dnf-plugins-core
+   $ sudo -E VERSION_ID=$VERSION_ID dnf config-manager --add-repo http://download.opensuse.org/repositories/home:/katacontainers:/release/Fedora\_$VERSION_ID/home:katacontainers:release.repo
+   $ sudo -E dnf -y install kata-runtime kata-proxy kata-shim
+   ```
 
-2. Install the Kata Containers components with the following commands:
+2. Decide which container manager you wish to use and follow the corresponding link below:
```

Comment:
Suggested rewrite:
2. Decide which container manager you will use and select the corresponding link that follows: